hail/python/test/hail/methods/test_qc.py

```diff
@@ -318,9 +318,10 @@ def test_charr(self):
         assert pytest.approx(d['C1046::HG02024'], abs=0.0001) == .00126
         assert pytest.approx(d['C1046::HG02025'], abs=0.0001) == .00124
 
+    @qobtest
     @skip_unless_service_backend(clouds=['gcp'])
     @set_gcs_requester_pays_configuration(GCS_REQUESTER_PAYS_PROJECT)
-    @test_timeout(batch=5 * 60)
+    @test_timeout(batch=10 * 60)
```

Comment:
did these tests get slower?

Reply:
No. I kept getting unlucky with the cluster being oversubscribed when I was trying to test this. I can revert back if you'd like.